internal/handler/registry.go

```diff
@@ -24,7 +25,7 @@ const (
 )
 
 func HandleClusterExtension(_ context.Context, fsys fs.FS, ext *ocv1alpha1.ClusterExtension) (*chart.Chart, chartutil.Values, error) {
-	plainFS, err := convert.RegistryV1ToPlain(fsys, ext.Spec.InstallNamespace, nil)
+	plainFS, err := convert.RegistryV1ToPlain(fsys, ext.Spec.InstallNamespace, []string{metav1.NamespaceAll})
```

Comment:
It looks like we no longer needs this fix because we no longer have it in operator-controller since #928